Dockerfile

```diff
@@ -25,6 +25,7 @@ ENV \
     BACKUP_CRON="0 */6 * * *" \
     RESTIC_FORGET_ARGS="" \
     RESTIC_JOB_ARGS=""
```

Comment:
Looks like a backslash `\` is missing

Actually we could also split up in separate ENV lines again, since it does not create more layers in Docker. I'm open to both :)

Reply:
Sorry, I missed that. Fixed it with https://github.com/lobaro/restic-backup-docker/pull/32/commits/4a1a5519235e91b856f94f4d89c9b8da71ea94fe.